config/katello-installer.yaml

```diff
@@ -45,6 +45,9 @@
     :foreman::plugin::templates:
       :dir_name: foreman
       :manifest_name: plugin/templates
+    :foreman::plugin::remote_execution:
+      :dir_name: foreman
+      :manifest_name: plugin/remote_execution
```

Comment:
You can add ```:params_name: plugin/remote_execution/params``` (see https://github.com/theforeman/foreman-installer/blob/42521b79e4e4a7a05dbf30b6fd5a2b783fb2e871/config/foreman-installer.yaml#L133-L136 for an example)